frontend/src/assets/projects.js

```diff
@@ -26,61 +26,61 @@ const projects = [
     websiteLink: 'http://volontuloapp.org/o'
   },
   {
-    description: '',
-    githubLink: '',
+    description: 'W wyniku współpracy z Polską Akcją Humanitarną stworzyliśmy aplikację pomagającą w logistycznych procesach akcji humanitarnych w różnych częściach globu. Aktualnie aplikacja służąca rejestracji i ewidencji przejazdów jest wykorzystywana podczas akcji humanitarnych na Ukrainie, w Sudanie Południowym, Jemenie oraz Somalii.',
+    githubLink: 'https://github.com/CodeForPoznan/pah-fm',
     image: '',
     license: '',
     licensePage: '',
     name: 'Fleet Manager',
-    partner: '',
-    partnerPage: '',
+    partner: 'PAH - Polska Akcja Humanitarna',
+    partnerPage: 'https://www.pah.org.pl/',
     stack: [
       {
         type: 'Frontend',
-        name: '',
+        name: 'JavaScript',
         version: '',
         documentation: ''
       },
       {
         type: 'Backend',
-        name: '',
+        name: 'Python',
         version: '',
         documentation: ''
       }
     ],
     website: ''
   },
   {
-    description: '',
-    githubLink: '',
+    description: ' Aplikacja Alinka powstała w celu optymalizacji codziennej pracy w poradniach psychologiczno-pedagogicznych. Jako byli pracownicy takiej poradni doskonale rozumiemy jakiego nakładu pracy wymaga wydanie orzeczenia czy sporządzenie protokołu z posiedzenia komisji. Aplikacja Alinka jest odpowiedzią na trudności z jakimi spotykają się w codziennej pracy pracownicy administracji poradni. Nazwa aplikacji: „Alinka” to imię prawdziwej osoby, która musi wykonywać tę nudną, ale bardzo ważną pracę. Mamy nadzieję, że któregoś dnia ta aplikacja pomoże w jej pracy. :)',
```

Comment:
Redundant space in the beginnig.

Reply:
(...) jest odpowiedzią na **trudności, z** jakimi spotykają się (...)

add comma